public/stylesheets/site/2.0/27-media-aural.css

```diff
@@ -74,9 +74,3 @@ code, kbd, tt, samp {
 .splash .browse li a:before {
   speak: none;
 }
-
-/*SAY-INSTEAD DRAFTS 
```

Comment:
This was basically a proposed CSS property that is no longer a thing. I removed reference to it from our docs, so I'm removing this comment as well.